ark/spLDA/processing.py

```diff
@@ -1,56 +1,45 @@
-import functools
-import matplotlib.pyplot as plt
-import numpy as np
 import pandas as pd
-import os
-import pickle
-from ark.utils.spatial_lda_utils import check_format_cell_table_args
-# from spatial_lda.featurization import *
-# from spatial_lda.visualization import plot_adjacency_graph()
+from sklearn.model_selection import train_test_split
+from spatial_lda.featurization import *
 
+from ark.settings import BASE_COLS
+from ark.utils.spatial_lda_utils import check_format_cell_table_args, \
+    check_featurize_cell_table_args
+
+train_frac = 0.75
```

Comment:
spooky global.  Looks like it can be made into a defaulted argument for `featurize_cell_table`.  Either way if there isn't a work around, it should be capitalized.